crates/core/tedge/src/cli/refresh_bridges.rs

```diff
@@ -0,0 +1,91 @@
+use camino::Utf8PathBuf;
+use tedge_config::TEdgeConfig;
+use tedge_config::TEdgeConfigLocation;
+
+use super::common::Cloud;
+use super::connect::ConnectError;
+use crate::bridge::BridgeConfig;
+use crate::bridge::CommonMosquittoConfig;
+use crate::bridge::TEDGE_BRIDGE_CONF_DIR_PATH;
+use crate::command::BuildContext;
+use crate::command::Command;
+
+pub struct RefreshBridgesCmd {
+    config: TEdgeConfig,
+    config_location: TEdgeConfigLocation,
+}
+
+impl Command for RefreshBridgesCmd {
+    fn description(&self) -> String {
+        "Refresh all currently active mosquitto bridges".to_string()
+    }
+
+    fn execute(&self) -> anyhow::Result<()> {
+        let clouds = established_bridges(&self.config_location);
+
+        if clouds.is_empty() {
+            println!("No bridges to refresh.");
```

Comment:
Return from here as there is no point proceeding when there are no bridges to refresh?

Reply:
fixed